canary/producer-c/canary/KvsProducerSampleCloudwatch.cpp

```diff
@@ -330,7 +349,12 @@ INT32 main(INT32 argc, CHAR* argv[])
                 lastKeyFrameTimestamp = frame.presentationTs;
             }
             CHK_STATUS(putKinesisVideoFrame(streamHandle, &frame));
-
+            if(firstFrame) {
```

Comment:
spacing after if (

Reply:
Could you add some comments about firstFrame? Why do we measure it here instead of before the loop? I mean, we do create in a sync fashion right? What's the delta between outputting here and right after the createsync succeeded? There will be a few hundreds of ns correct?

Reply:
That is correct. Added a comment to explain why it is measured here